spec/models/event_spec.rb

```diff
@@ -167,5 +167,17 @@
     application_letter_accepted_2 = FactoryGirl.create(:application_letter_accepted, user: FactoryGirl.create(:user), event: event)
     expect(event.compute_occupied_places).to eq(2)
   end
+
+  it "generates an email list" do
+    event = FactoryGirl.create(:event)
+    application_letter = FactoryGirl.create(:application_letter, user: FactoryGirl.create(:user), event: event)
+    application_letter_accepted = FactoryGirl.create(:application_letter_accepted, user: FactoryGirl.create(:user), event: event)
+    event.application_letters.push(application_letter)
+    event.application_letters.push(application_letter_accepted)
```

Comment:
Since FactoryGirl saves the ApplicationLetters, they get automatically added to the Event, so lines 175-176 and 179 are unnecessary.